versions.yaml

```diff
@@ -44,3 +44,8 @@ oci:
   pause:
     registry: docker://registry.k8s.io/pause
     tag: 3.6
+
+kbs:
+  image:
+    registry: ghcr.io/confidential-containers/key-broker-service
+    tag: v0.8.0
```

Comment:
Can we also add the kbs code version corresponding to the image? Like github and commit or revision?